main.js

```diff
@@ -0,0 +1,23 @@
+import oDate from './date';
+
+import layout from 'n-layout';
```

Comment:
yep. any non origami thing referenced in here should get moved in here once I'v eroled n-ui out everywhere